logging/processors.rst

```diff
@@ -164,6 +164,30 @@ If you use several handlers, you can also register a processor at the
 handler level or at the channel level instead of registering it globally
 (see the following sections).
 
+When registering a new processor, instead of adding the tag manually in your
+configuration files, you can take full advantage of the ``#[AsMonologProcessor]``
```

Comment:
as you target v5.4 and this can run on php v7, perhaps need to give hint that this only works with php v8+

Reply:
I would say yes, but it seems that all `versionadded` of the doc about attributes are not displaying this information, even on 5.4 🤔

Reply:
Indeed, we avoid the explicit PHP hint for PHP 8, but thanks for raising it 👌🏻